infrastructure/terraform/components/iam/module_amplify_branch.tf

```diff
@@ -15,5 +15,7 @@ module "amplify_branch" {
   branch                                    = "main"
   domain_name                               = local.root_domain_name
   subdomain                                 = var.environment
+  base_path                                 = "/auth"
   enable_auto_deploy                        = true
+  stage                                     = "PRODUCTION"
```

Comment:
should this be var.stage or is this 'default should be to deploy prod' in action?

Reply:
the main branch of the IAM component should be prod yea, it does this and seperates the branch in to a nice little group of its own:
![image](https://github.com/user-attachments/assets/718a5a65-cf81-4848-8712-d9368fb76974)